pkg/controller/v1beta1/inferenceservice/reconcilers/knative/ksvc_reconciler.go

```diff
@@ -150,6 +167,34 @@ func createKnativeService(
 		return !utils.Includes(disallowedLabelList, key)
 	})
 
+	if storageUris != nil && len(*storageUris) > 0 {
+		isvcReadonlyStringFlag := true
+		isvcReadonlyString, ok := annotations[constants.StorageReadonlyAnnotationKey]
+		if ok {
+			if isvcReadonlyString == "false" {
+				isvcReadonlyStringFlag = false
+			}
+		}
```

Comment:
nit: It's simple but it's repeated in three places. would it be better to have a util function?